web/src/pages/Administration/CountryList/countrylist.scss

```diff
@@ -0,0 +1,5 @@
+.Country_List_main_cls{
```

Comment:
Class name should be in camelcase or hyphen seperated
E.X. = redBox or red-box

Reply:
Updated